lib/web_ui/lib/src/engine/keyboard.dart

```diff
@@ -136,13 +141,15 @@ int _getMetaState(html.KeyboardEvent event) {
   if (event.getModifierState('Meta')) {
     metaState |= _modifierMeta;
   }
-  if (event.getModifierState('NumLock')) {
+  if (event.keyCode == _kNumLockKeyCode && event.getModifierState('NumLock')) {
```

Comment:
Right now, there are two ways the Flutter framework receives information about meta keys (alt, shift, numlock, etc):
1. Through the `metaState` flag which is included in all keyboard events.
2. Through the `keydown`/`keyup` event of the meta key itself (pressing the `NumLock` key triggers a `keydown` event and is sent to Flutter).

The issue we are seeing is because of (1). @gspencergoog does the framework really need both of these ways of sending meta keys?

Reply:
removed the modifier state for now.